docs/parameters.yaml

```diff
@@ -291,13 +291,6 @@ type: bool
 default: true
 components: ["origin"]
 ---
-name: Origin.EnableUI
```

Comment:
I think removing this is a backward compatibility issue -- what happens in places where this is already set?

It's worth a discussion with @haoming29 -- should the `EnableUI` flag really be just the UI (that is, the piece that is viewed in the browser) or all the API functionality.  If it's the latter, migrating to a common `Server.EnableUI` and deprecating the `Origin.EnableUI` flag is reasonable.  I suspect we might lean toward the former as the API components are becoming critical -- what is the registry without the APIs?

Reply:
I decided to re-introduce the `Origin.EnableUI` parameter but keep the references pointed to `Server.EnableUI`.

I would like the `Server.EnableUI` to refer to any browser-based interface whatsoever whereas the `$MODULE.EnableUI` would be only the relevant pane in the browser interface.